hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneRpcClientAbstract.java

```diff
@@ -4123,4 +4124,34 @@ private OzoneBucket getBucket(OzoneVolume volume) throws IOException {
   private static ReplicationConfig anyReplication() {
     return RatisReplicationConfig.getInstance(HddsProtos.ReplicationFactor.ONE);
   }
+
```

Comment:
assertBucketHasSnapshotCount and assertBucketCount can be a simple one function.
1.
```
  private void assertBucketCount(OzoneVolume volume,
                                 String bucketPrefix,
                                 boolean hasSnapshot,
                                 int expectedBucketCount) {
    Iterator<? extends OzoneBucket> bucketIterator;
    if (hasSnapshot) {
      bucketIterator = volume.listBuckets(bucketPrefix, null, hasSnapshot);
    } else {
      bucketIterator = volume.listBuckets(bucketPrefix);
    }

    int bucketCount = 0;
    while (bucketIterator.hasNext()) {
      Assert.assertTrue(
          bucketIterator.next().getName().startsWith(bucketPrefix));
      bucketCount++;
    }
    Assert.assertEquals(expectedBucketCount, bucketCount);
  }
 ```